src/locutus/model/terminology.py

```diff
@@ -572,6 +572,76 @@ def remove_pref(self, code=None):
 
         return message
 
+    def get_preferred_terminology(self):
+        """
+        Retrieves all references from the 'preferred_terminology' sub-collection
+
+        Returns:
+            dict: An array of `Terminology` references under the "references" key
+        
+        Example:
+        {
+            "reference": [

```

Comment:
Let's change this to 
[
  { 
     "reference": "Terminology/tm--asdfasdf"
  },
  { 
    "reference": "Terminology/tm-fdsa-fdsa"
  }
]

If you want to return an object, then you could name the array you return. Also, those reference strings will not start with a slash. It's just {ResourceType}/{id} -- blatant stealing from FHIR.